src/service/performance-impl.js

```diff
@@ -97,7 +97,7 @@ export class Performance {
         });
 
     // Tick window.onload event.
-    loadPromise(win).then(() => {
+    loadPromise(win.document).then(() => {
```

Comment:
The code now waits for `complete` state, right? Which should be the same as before.